addons/mail/static/src/models/thread_view_topbar.js

```diff
@@ -672,6 +672,11 @@ registerModel({
         pendingThreadDescription: attr({
             default: "",
         }),
+        shouldDisplayDebugIcon: attr({
+            compute() {
+                return this.threadView.threadViewer.discussPublicView && odoo.debug !== '';
```

Comment:
I've not found other occurrences of `odoo.debug` being tested as empty string in the rest of the code, it looks like we prefer casting it to a boolean.

Reply:
Initially, it was cast as Boolean instead of comparing it with an empty string. But, in a previous review @oomsveta suggested it to be like this. https://github.com/odoo/odoo/pull/97178#discussion_r971663624